lib/modules/dosomething/dosomething_campaign/dosomething_campaign_hot_shares/dosomething_campaign_hot_shares.cron.inc

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Provides cron job command for dosomething_campaign_problem_shares.module.
+ */
+
+/**
+ * Implements hook_cron().
+ */
+function dosomething_campaign_hot_shares_cron() {
+  $results = db_query("SELECT nid FROM node WHERE type ='campaign'");
+
+  // Get the relevant paths
+  $path = dosomething_helpers_application_path('/lib/modules/dosomething/dosomething_campaign/dosomething_campaign_hot_shares');
+  $font_path = dosomething_helpers_application_path('/lib/modules/dosomething/dosomething_campaign/fonts');
+  $image_location = dosomething_helpers_application_path('/html/sites/default/files');
+
+  // Each active campaign, call shell script to create problem share images.
+  foreach($results as $result) {
+    $campaign = Campaign::get($result->nid)[0];
+    $node = node_load($result->nid);
```

Comment:
I'm curious here, since we are essentially get the campaign twice here, (one is a `node_load` and the other is with the class), which takes longer, and if we could squish this into only one way of getting the node

we could either attach `time_left` onto the campaign class, or get the reportback `noun & verb` from the sql query

my worry is that we have a lot going on in this cron and I don't want it to be super slow, but maybe that's an unfounded worry